src/gui/src/components/queries/queries-empty-state.tsx

```diff
@@ -3,12 +3,42 @@ import { Button } from '@/components/ui/button.jsx'
 import { Command, Plus, Star } from 'lucide-react'
 import { AnimatePresence, motion, useAnimate } from 'framer-motion'
 import { CreateQuery } from '@/components/queries/create-query.jsx'
+import { useGraphStore } from '@/state/index.js'
+import { startDashboardData } from '@/lib/start-dashboard-data.js'
 
 const QueriesEmptyState = () => {
   const [starScope, starAnimate] = useAnimate<HTMLDivElement>()
   const [cursorScope, cursorAnimate] = useAnimate<HTMLDivElement>()
   const [isCreating, setIsCreating] = useState<boolean>(false)
 
+  const dashboard = useGraphStore(state => state.dashboard)
+  const updateActiveRoute = useGraphStore(
+    state => state.updateActiveRoute,
+  )
+  const updateDashboard = useGraphStore(
+    state => state.updateDashboard,
+  )
+  const updateErrorCause = useGraphStore(
+    state => state.updateErrorCause,
+  )
+  const stamp = useGraphStore(state => state.stamp)
+
+  useEffect(() => {
+    startDashboardData({
+      updateActiveRoute,
+      updateDashboard,
+      updateErrorCause,
+      stamp,
+    })
```

Comment:
Let's make sure to pass down from the `app/queries.tsx` component that already have this data.